### ISSUE TITLE
[BUG] Security modules ignore plugins.security.ssl.transport.pemtrustedcas_filepath

### ISSUE THREAD

**Issue by patcable** [opensearch-project/security#1866]
**What is the bug?**
When I attempt to authenticate to an LDAP server with `plugins.security.ssl.transport.pemtrustedcas_filepath` set, it's unable to validate the LDAP server's TLS certificate.

**How can one reproduce the bug?**
1. Set PEM Trusted CAs.
2. Try to authenticate against an LDAP server

**What is the expected behavior?**
The security module would use whatever trusted CA option is set to validate the TLS connection

**What is your host/environment?**
 - OS: Ubuntu Linux 20.04
 - Version 2.0.0

**Do you have any screenshots?**
This stack trace may be helpful:
```
[2022-06-01T16:54:43,991][DEBUG][c.a.d.a.l.b.LDAPAuthenticationBackend] [i-055541de785b4851d] Unable to authenticate user due to
org.ldaptive.LdapException: Unable to connect to any of those ldap servers [ldap.REMOVED:389] due to OpenSearchException[Empty file path for plugins.security.ssl.transport.truststore_filepath]
	at com.amazon.dlic.auth.ldap.backend.LDAPAuthorizationBackend.getConnection0(LDAPAuthorizationBackend.java:364) ~[opensearch-security-2.0.0.0.jar:2.0.0.0]
	at com.amazon.dlic.auth.ldap.backend.LDAPAuthorizationBackend$2.run(LDAPAuthorizationBackend.java:169) ~[opensearch-security-2.0.0.0.jar:2.0.0.0]
	at com.amazon.dlic.auth.ldap.backend.LDAPAuthorizationBackend$2.run(LDAPAuthorizationBackend.java:159) ~[opensearch-security-2.0.0.0.jar:2.0.0.0]
	at java.security.AccessController.doPrivileged(AccessController.java:569) ~[?:?]
	at com.amazon.dlic.auth.ldap.backend.LDAPAuthorizationBackend.getConnection(LDAPAuthorizationBackend.java:159) ~[opensearch-security-2.0.0.0.jar:2.0.0.0]
	at com.amazon.dlic.auth.ldap.backend.LDAPAuthenticationBackend.authenticate(LDAPAuthenticationBackend.java:87) [opensearch-security-2.0.0.0.jar:2.0.0.0]
	at org.opensearch.security.auth.BackendRegistry$5.call(BackendRegistry.java:496) [opensearch-security-2.0.0.0.jar:2.0.0.0]
	at org.opensearch.security.auth.BackendRegistry$5.call(BackendRegistry.java:490) [opensearch-security-2.0.0.0.jar:2.0.0.0]
	at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4853) [guava-30.0-jre.jar:?]
	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3529) [guava-30.0-jre.jar:?]
	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2278) [guava-30.0-jre.jar:?]
	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2155) [guava-30.0-jre.jar:?]
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2045) [guava-30.0-jre.jar:?]
	at com.google.common.cache.LocalCache.get(LocalCache.java:3951) [guava-30.0-jre.jar:?]
	at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4848) [guava-30.0-jre.jar:?]
	at org.opensearch.security.auth.BackendRegistry.authcz(BackendRegistry.java:490) [opensearch-security-2.0.0.0.jar:2.0.0.0]
	at org.opensearch.security.auth.BackendRegistry.authenticate(BackendRegistry.java:299) [opensearch-security-2.0.0.0.jar:2.0.0.0]
	at org.opensearch.security.filter.SecurityRestFilter.checkAndAuthenticateRequest(SecurityRestFilter.java:192) [opensearch-security-2.0.0.0.jar:2.0.0.0]
	at org.opensearch.security.filter.SecurityRestFilter$1.handleRequest(SecurityRestFilter.java:125) [opensearch-security-2.0.0.0.jar:2.0.0.0]
	at org.opensearch.rest.RestController.dispatchRequest(RestController.java:311) [opensearch-2.0.0.jar:2.0.0]
	at org.opensearch.rest.RestController.tryAllHandlers(RestController.java:397) [opensearch-2.0.0.jar:2.0.0]
	at org.opensearch.rest.RestController.dispatchRequest(RestController.java:240) [opensearch-2.0.0.jar:2.0.0]
	at org.opensearch.security.ssl.http.netty.ValidatingDispatcher.dispatchRequest(ValidatingDispatcher.java:63) [opensearch-security-2.0.0.0.jar:2.0.0.0]
	at org.opensearch.http.AbstractHttpServerTransport.dispatchRequest(AbstractHttpServerTransport.java:366) [opensearch-2.0.0.jar:2.0.0]
	at org.opensearch.http.AbstractHttpServerTransport.handleIncomingRequest(AbstractHttpServerTransport.java:445) [opensearch-2.0.0.jar:2.0.0]
	at org.opensearch.http.AbstractHttpServerTransport.incomingRequest(AbstractHttpServerTransport.java:356) [opensearch-2.0.0.jar:2.0.0]
	at org.opensearch.http.netty4.Netty4HttpRequestHandler.channelRead0(Netty4HttpRequestHandler.java:55) [transport-netty4-client-2.0.0.jar:2.0.0]
	at org.opensearch.http.netty4.Netty4HttpRequestHandler.channelRead0(Netty4HttpRequestHandler.java:41) [transport-netty4-client-2.0.0.jar:2.0.0]
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at org.opensearch.http.netty4.Netty4HttpPipeliningHandler.channelRead(Netty4HttpPipeliningHandler.java:71) [transport-netty4-client-2.0.0.jar:2.0.0]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) [netty-codec-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) [netty-codec-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) [netty-codec-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:327) [netty-codec-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:299) [netty-codec-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286) [netty-handler-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) [netty-codec-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1371) [netty-handler-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1234) [netty-handler-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1283) [netty-handler-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:510) [netty-codec-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:449) [netty-codec-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:279) [netty-codec-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:722) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysPlain(NioEventLoop.java:623) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:586) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496) [netty-transport-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986) [netty-common-4.1.73.Final.jar:4.1.73.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.73.Final.jar:4.1.73.Final]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: org.opensearch.OpenSearchException: Empty file path for plugins.security.ssl.transport.truststore_filepath
	at org.opensearch.security.support.PemKeyReader.checkPath(PemKeyReader.java:263) ~[opensearch-security-2.0.0.0.jar:2.0.0.0]
	at org.opensearch.security.support.PemKeyReader.resolve(PemKeyReader.java:336) ~[opensearch-security-2.0.0.0.jar:2.0.0.0]
	at org.opensearch.security.support.PemKeyReader.resolve(PemKeyReader.java:322) ~[opensearch-security-2.0.0.0.jar:2.0.0.0]
	at com.amazon.dlic.auth.ldap.backend.LDAPAuthorizationBackend.configureSSL(LDAPAuthorizationBackend.java:564) ~[opensearch-security-2.0.0.0.jar:2.0.0.0]
	at com.amazon.dlic.auth.ldap.backend.LDAPAuthorizationBackend.getConnection0(LDAPAuthorizationBackend.java:287) ~[opensearch-security-2.0.0.0.jar:2.0.0.0]
	... 81 more
```

**Do you have any additional context?**
Super thankful for y'all getting logging working #1689 in so I could uncover this bug 😄 

I'm happy to set up a java-flavored truststore and such to move past this, but it is confusing behavior.

**Comment by cliu123** [comment]
[Triage] Thanks for reporting the issue!

**Comment by GadskyPapa** [comment]
There is the same trouble using keycloak. I've added to ca.crt a self-signed CA of my opensearch cluster and Digicert's CA of the cert of keycloak.domain.com. And it doesn't work, look at the error below. Then I took cacerts from an embedded java /usr/share/opensearch/jdk/lib/security/cacerts and put it to the opensearch config directory. And It works. I use the official docker image v 2.1.0.

from opensearch.yml:
...
plugins.security.ssl.transport.pemtrustedcas_filepath: ca.crt
plugins.security.ssl.transport.truststore_filepath: cacerts

[2022-07-19T17:25:24,604][ERROR][o.o.s.s.DynamicConfigModelV7] [os-data1] Unable to initialize auth domain openid_auth_domain=AuthcDomain [http_enabled=true, transport
_enabled=true, order=2, http_authenticator=HttpAuthenticator [challenge=false, type=openid, config={openid_connect_idp={enable_ssl=true, verify_hostnames=false}, subject_key=preferred_username, roles_key=roles, openid_connect_url=https://keycloak.domain.com/realms/master/.well-known/openid-configuration}], authentication_backend=
AuthcBackend [type=noop, config={}], description=null] due to OpenSearchException[java.lang.reflect.InvocationTargetException]; nested: InvocationTargetException; nested: RuntimeException[com.amazon.dlic.util.SettingsBasedSSLConfigurator$SSLConfigException: Error loading trust store from null]; nested: SSLConfigException[Error loading trust store from null]; nested: OpenSearchException[Empty file path for plugins.security.ssl.transport.truststore_filepath];

**Comment by peternied** [comment]
@GadskyPapa could you paste more of your configuration .yml, please redacat as needed, it seems like there might be additional properties that need to be assigned for OpenID Connect.  Could you take a look at https://opensearch.org/docs/latest/security-plugin/configuration/openid-connect/#certificate-validation and see if that unblocks using keycloak?

**Comment by GadskyPapa** [comment]
> @GadskyPapa could you paste more of your configuration .yml, please redacat as needed, it seems like there might be additional properties that need to be assigned for OpenID Connect. Could you take a look at https://opensearch.org/docs/latest/security-plugin/configuration/openid-connect/#certificate-validation and see if that unblocks using keycloak?

I have the same error "Unable to initialize auth domain openid_auth_domain=AuthcDomain ..." without "plugins.security.ssl.transport.truststore_filepath: cacerts" in opensearch.yml. And it doesn't matter is there "pemtrustedcas_filepath" in opensearch-security/config.yml or not.

opensearch.yml
===================================================
cluster.name: opensearch-test
node.name: os-data1
network.host: 0.0.0.0
transport.host: 0.0.0.0
#node.roles: []
discovery.seed_hosts: ["os-data1","os-data2","os-data3"]
cluster.initial_master_nodes: ["os-data1","os-data2","os-data3"]

plugins.security.ssl.transport.pemcert_filepath: os-data1.crt
plugins.security.ssl.transport.pemkey_filepath: os-data1.key
plugins.security.ssl.transport.pemtrustedcas_filepath: ca.crt
plugins.security.ssl.transport.enforce_hostname_verification: false
#plugins.security.ssl.transport.truststore_filepath: cacerts
plugins.security.ssl.transport.enabled_protocols:
  - "TLSv1.2"
  - "TLSv1.3"
plugins.security.ssl.http.enabled: true
plugins.security.ssl.http.clientauth_mode: OPTIONAL
plugins.security.ssl.http.pemcert_filepath: os-data1.crt
plugins.security.ssl.http.pemkey_filepath: os-data1.key
plugins.security.ssl.http.pemtrustedcas_filepath: ca.crt
plugins.security.ssl.http.enabled_protocols:
  - "TLSv1.2"
  - "TLSv1.3"
plugins.security.allow_default_init_securityindex: false
plugins.security.authcz.admin_dn:
  - "CN=os-admin,OU=client,O=client,L=test,C=de"
plugins.security.nodes_dn:
  - "CN=os-data*,OU=client,O=client,L=test,C=de"
plugins.security.audit.type: internal_opensearch
plugins.security.enable_snapshot_restore_privilege: true
plugins.security.check_snapshot_restore_write_privileges: true
plugins.security.restapi.roles_enabled: ["all_access", "security_rest_api_access"]
plugins.security.allow_unsafe_democertificates: false
===================================================

opensearch-security/config.yml
===================================================
_meta:
  type: "config"
  config_version: 2
config:
  dynamic:
    # Set filtered_alias_mode to 'disallow' to forbid more than 2 filtered aliases per index
    # Set filtered_alias_mode to 'warn' to allow more than 2 filtered aliases per index but warns about it (default)
    # Set filtered_alias_mode to 'nowarn' to allow more than 2 filtered aliases per index silently
    #filtered_alias_mode: warn
    #do_not_fail_on_forbidden: false
    kibana:
    # Kibana multitenancy
      multitenancy_enabled: true
      server_username: kibanaserver
      index: '.kibana'

    authc:
      openid_auth_domain:
        http_enabled: true
        transport_enabled: true
        order: 1
        http_authenticator:
          type: openid
          challenge: false
          config:
            openid_connect_idp:
              enable_ssl: true
              pemtrustedcas_filepath: /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
              verify_hostnames: false
            subject_key: preferred_username
            roles_key: roles
            openid_connect_url: https://keycloak.mydomain.com/realms/master/.well-known/openid-configuration
        authentication_backend:
          type: noop

      clientcert_auth_domain:
        description: "Authenticate via SSL client certificates"
        http_enabled: true
        transport_enabled: true
        order: 0
        http_authenticator:
          type: clientcert
          config:
            username_attribute: cn #optional, if omitted DN becomes username
          challenge: false
        authentication_backend:
          type: noop
	basic_internal_auth_domain:
        http_enabled: true
        transport_enabled: true
        order: 2
        http_authenticator:
          type: basic
          challenge: false
        authentication_backend:
          type: intern

**Comment by peternied** [comment]
It looks like `pemtrustedcas_content` needs to be added to get this working [1], can you make an update like the following and see if this gets things working?

```yml
            openid_connect_idp:
              enable_ssl: true
              pemtrustedcas_content: {{ PATH TO CERTIFICATE }}
              pemtrustedcas_filepath: /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
              verify_hostnames: false
            subject_key: preferred_username
            roles_key: roles
            openid_connect_url: https://keycloak.mydomain.com/realms/master/.well-known/openid-configuration
```


[1] https://github.com/opensearch-project/security/blob/main/src/main/java/com/amazon/dlic/util/SettingsBasedSSLConfigurator.java#L181

**Comment by pawelw1** [comment]
@GadskyPapa Have you tried moving tls-ca-bundle.pem file to the `/usr/share/opensearch/config/` folder? 
@peternied  `pemtrustedcas_content` requires the content of the .pem file and not the file path. 
i.e. 

```
pemtrustedcas_content: |-
              -----BEGIN CERTIFICATE-----
            
            
            #      Certificate content
            
            
              -----END CERTIFICATE-----
```

**Comment by scrawfor99** [comment]
[Triage] Closing this issue since it seems like it is resolved. Please open a new issue for any related bugs.

**Comment by vaskosmihaylov** [comment]
I am afraid this issue is not resolved. I recently tried to implement LDAP with Openseach and experienced the same problem. if the `plugins.security.ssl.transport.truststore_filepath` is missing from the `opensearch.yml`  the following error is observed in the logs:

```[2023-03-06T16:59:20,889][WARN ][c.a.d.a.l.b.LDAPAuthorizationBackend] [vasko-node-1] Unable to connect to ldapserver host.come:636 due to OpenSearchException[Empty file path for plugins.security.ssl.transport.truststore_filepath]. Try next.```

I tried to set the `pemtrustedcas_filepath` and `pemtrustedcas_content` in the `config.yml` none of them worked.  If you need more information I am happy to provide it.

**Comment by pawelw1** [comment]
@vaskosmihaylov Could you open a new thread in OpenSearch [forum](https://forum.opensearch.org/c/security/3)?
Please share your config.yml, opensearch.yml and version of the OpenSearch.
